src/sass/essay/_base.scss

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2018 The Trustees of Indiana University
+// SPDX-License-Identifier: BSD-3-Clause
+
+@use 'core' as *;
+
+.rvt-essay {
+  display: flex;
+  flex-direction: column;
+
+  &__column:focus-within {
+    z-index: $z-index-100;
+  }
+
+  &__stat {
+    align-items: center;
+    background-color: $color-cream-base;
+    display: flex;
+    flex-direction: column;
+    height: $width-md;
+    justify-content: center;
+    padding: $spacing-sm;
+    text-decoration: none;
+  }
+
+  &__stat-number {
+    font-size: $ts-46;
+  }
+
+  &__stat-caption {
+    margin-bottom: 0;
+  }
+
+  &__stat-number,
+  &__stat-caption {
+    color: $color-crimson-base;
+    font-weight: $font-weight-bold;
+  }
+
+  .rvt-bg-crimson &__stat-number,
+  .rvt-bg-crimson &__stat-caption,
+  .rvt-bg-crimson-dark &__stat-number,
+  .rvt-bg-crimson-dark &__stat-caption {
+    color: $color-white-base;
+  }
+
+  &__image {
+    background-color: $color-crimson-base;
+    background-position: center center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    display: block;
+    height: $width-md;
+  }
+}
+
+a.rvt-essay {
+  &__stat:hover,
+  &__image:hover {
+    opacity: .5;
+    transition: all .3s linear;
+  }
+}
+
+@media screen and (min-width: 800px) {
```

Comment:
Let's use a variable for this value here instead of hard coding. Eg. `$breakpoint-md`. The `800px` value in the prototype wasn't chosen with any purpose, so we should pick one of our predefined breakpoints, unless there is a specific need for it to be 800px.